### PR TITLE
Fix #161: pksetup assert versions

### LINKED ISSUE #161: assert versions in pksetup
setuptools is often broken so assert the versions we expect in pksetup.py so we can fail in a clean way not like https://github.com/radiasoft/devops/issues/265

### PR DESCRIPTION
**Pull request by gurhar1133**
* _assert_package_versions() ensures setuptools version is correct
* Some imports changed in pkcli/test.py so that unknown options in pykern test invocations throws correct error

### CODE REVIEW

File: pykern/pksetup.py [hunk 1]
```diff
@@ -342,7 +342,18 @@ def main():
...
+        """Raise assertion if another module has installed incompatible version of setuptools
+        """
+        from packaging.version import Version
+
+        assert Version(setuptools.__version__) < Version('57'), \
+            'Some package installed a newer version of setup tools that does not work with pykern.pksetup'
+
     name = kwargs['name']
+    if name != 'pykern':
+        # POSIT: setup.py has 'setuptools<57'
```
Comment (robnagler) [inline pykern/pksetup.py:355]: This comment should be moved to _assert_package_versions docstring.

**Review by robnagler — CHANGES_REQUESTED** [review]

File: pykern/pksetup.py (hunk 1 above)
Comment (gurhar1133) [reply pykern/pksetup.py:355]: Just move it into the docstring like this
```python
 def _assert_package_versions():
        """Raise assertion if another module has installed incompatible version of setuptools
            POSIT: setup.py has 'setuptools<57'
        """
```
or replace the docstring? ei.
```python
 def _assert_package_versions():
        """POSIT: setup.py has 'setuptools<57'"""
```

Comment (robnagler) [reply pykern/pksetup.py:355]: Move the string and add some space and formatting:

```
"""Raise assertion if another module has installed incompatible version of setuptools

POSIT: setup.py has 'setuptools<57'
""""
```

Comment (gurhar1133) [reply pykern/pksetup.py:355]: You got it 👍

**Comment by gurhar1133** [comment]
Made the change @robnagler

**Review by robnagler — APPROVED** [review]

**Pull request merged**